<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmDeviceConfig</class>
 <widget class="QMainWindow" name="FrmDeviceConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240,240, 240);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>530</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>105</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(50, 150, 200);</string>
    </property>
    <property name="text">
     <string>Thời gian lấy mẫu</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_sampleTime">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_setSampleTime">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
color: rgb(255, 255, 255);
background-color: rgb(50, 150, 200);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 6px;
}
QToolButton:hover{
background-color: rgb(120, 210, 240);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Cài đặt</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextOnly</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_zeroText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>55</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(50, 150, 200);</string>
    </property>
    <property name="text">
     <string>Thiết lập điểm không tại</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_setZero">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
color: rgb(255, 255, 255);
background-color: rgb(50, 150, 200);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 6px;
}
QToolButton:hover{
background-color: rgb(120, 210, 240);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Zero</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextOnly</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>530</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(2,87, 122);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_Name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>6</y>
       <width>431</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Cảm biến</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>530</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_clearZero">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
color: rgb(255, 255, 255);
background-color: rgb(50, 150, 200);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 6px;
}
QToolButton:hover{
background-color: rgb(120, 210, 240);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Xóa</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextOnly</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>156</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(50, 150, 200);</string>
    </property>
    <property name="text">
     <string>Độ phân giải</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_unit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
color: rgb(2, 87, 122);
background-color: rgb(255, 255, 255);
border: none;
border-style: outset;
border-width: 0px;
border-top-left-radius: 15px;
border-bottom-left-radius: 15px;
}

QComboBox::drop-down{
width: 20;
}
QComboBox:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>Độ ẩm tương đối (%RH)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Độ ẩm tuyệt đối (g/m3)</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_resolution">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
color: rgb(2, 87, 122);
background-color: rgb(255, 255, 255);
border: none;
border-style: outset;
border-width: 0px;
border-top-left-radius: 15px;
border-bottom-left-radius: 15px;
}
QComboBox::drop-down{
width: 20;
}
QComboBox:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.01</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.001</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>156</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(50, 150, 200);</string>
    </property>
    <property name="text">
     <string>Đơn vị đo</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>530</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Zero">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Unit">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>57</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Unit_2">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>107</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>(ms)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_tempCompensation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(50, 150, 200);</string>
    </property>
    <property name="text">
     <string>Bù nhiệt độ cho phép đo</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
